source/PhysicalQuantity/Force.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace PhpUnitsOfMeasure\PhysicalQuantity;
+
+use PhpUnitsOfMeasure\AbstractPhysicalQuantity;
+use PhpUnitsOfMeasure\UnitOfMeasure;
+use PhpUnitsOfMeasure\HasSIUnitsTrait;
+
+class Force extends AbstractPhysicalQuantity
+{
+	use HasSIUnitsTrait;
+
+	protected static $unitDefinitions;
+
+	protected static function initialize()
+	{
+
+		// newton
+		$newton = UnitOfMeasure::nativeUnitFactory('N');
+		$newton->addAlias('newton');
+		static::addUnit($newton);
+
+		static::addMissingSIPrefixedUnits(
+			$newton,
+			1,
+			'%pN',
+			[
+				'%Pnewton',
+				'%Pnewtons'
+			]
+		);
+
+		// kilonewton
+		$newUnit = UnitOfMeasure::linearUnitFactory('kN', 1000);
```

Comment:
you won't need to add this kilonewton section explicitly; the `static:addMissingSIPrefixedUnits()` call above will handle adding all the standard SI prefixes like `mili`, `mega`, `kilo`, etc.